added: files of new website

kausarpial opened the pull request:
### Description

- [x] Added all the file of new website

nsorros commented:
@kausarpial This is super 👍 Some questions

- Does this add a Github action to automatically deploy when changes are made? By deploy I mean rebuild the website.
- Was Netlify support to work cause I cannot see a preview of the website

kausarpial commented:
> @kausarpial This is super 👍 Some questions
> 
> * Does this add a Github action to automatically deploy when changes are made? By deploy I mean rebuild the website.
> * Was Netlify support to work cause I cannot see a preview of the website
> * Can you include some instructions in the README in a development section for rebuilding the website and updating the blog part? Some information on the structure of the project and how we add content is also useful.

Hi @nsorros 
- We don't need to add any Github action to deploy the website when any changes are made. Netlify will automatically detect the changes and deploy that for us. 
- Yes, we need some changes on the Netlify deployment to see the preview. But your current deployment doesn't need any configuration on Netlify. For that, we can create a new site on Netlify and see the previews on that. After we're satisfied with the result, we can merge the merge onto the main. Then we can update the netlify configuration for deployment. 
![Screenshot 2022-12-15 at 12 55 22 PM](https://user-images.githubusercontent.com/16266381/207845178-039eaea5-5ec8-40df-a9c7-d7d4d19b7025.png)
- Sure, I'll add some documentation on the Readme file to get an easy grasp of the project. 

Thanks

nsorros commented:
> Also, is it possible to add me to your Netlify team? Thanks
@ivyleavedtoadflax could you resolve that?

>     * Yes, we need some changes on the Netlify deployment to see the preview. But your current deployment doesn't need any configuration on Netlify. For that, we can create a new site on Netlify and see the previews on that. After we're satisfied with the result, we can merge the merge onto the main. Then we can update the netlify configuration for deployment.
Let me know if you need anything else from us to get the preview up so we can approve and merge :)

ivyleavedtoadflax commented:
> > > Also, is it possible to add me to your Netlify team? Thanks
> > > @ivyleavedtoadflax could you resolve that?
> > 
> > 
> > > ```
> > > * Yes, we need some changes on the Netlify deployment to see the preview. But your current deployment doesn't need any configuration on Netlify. For that, we can create a new site on Netlify and see the previews on that. After we're satisfied with the result, we can merge the merge onto the main. Then we can update the netlify configuration for deployment.
> > > ```
> > 
> > 
> > Let me know if you need anything else from us to get the preview up so we can approve and merge :)
> 
> Sure, Thanks @nsorros Here's my Nelify email: [kausarahmedpial@gmail.com](mailto:kausarahmedpial@gmail.com)

Done

kausarpial commented:
Hi @nsorros @ivyleavedtoadflax 
I've made some changes on netlify to build, and also updated the readme file. 
Now, the deploy preview will work. 

Let me know of further changes. 
Thanks

nsorros commented:
> Hi @nsorros @ivyleavedtoadflax I've made some changes on netlify to build, and also updated the readme file. Now, the deploy preview will work.
> 
> Let me know of further changes. Thanks

We need to remove the first item in work and change the title of the second one according to whats live atm. Should I try to do the change for practice?

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNhMDA1YWI0MTgwNDcwMDA5YTk0ZDk2IiwiaWQiOiI0NWQwMTg1NS03NjQ4LTRkNmUtYjVlNC1jOTE5YjJiYTMzNTUifQ==)

ivyleavedtoadflax commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNhMDA1YWI0MTgwNDcwMDA5YTk0ZDk2IiwiaWQiOiIxMjYzYjI3OC03MDU0LTRmYWUtODQ1NC1jYmJjZjYzZjRmY2QifQ==)

This should be we `We believe` or `Our Beliefs` @nsorros ?


<details><summary>Browser metadata</summary>





```
Path:      /about/
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2Fabout%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/about/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a02790fceb144172a7f995&resolution=resolved)

ivyleavedtoadflax commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNhMDA1YWI0MTgwNDcwMDA5YTk0ZDk2IiwiaWQiOiJjYTU0MDQ4Ni1iOTgzLTQ0YWUtOGI4Yy0zNTdlYTlhYWQ4ZGQifQ==)

Let's change this for something else, I don't think it works well at this scale. Or maybe we can just make it smaller?

<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a027cc57a0d33e8e1b5cfd&resolution=resolved)

ivyleavedtoadflax commented:
More a content question, but perhaps we should change this to be more general - less medical specific @nsorros ![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNhMDA1YWI0MTgwNDcwMDA5YTk0ZDk2IiwiaWQiOiJiMGQxODcyMS03MDk0LTRmZWUtOTZhZS01Mjc4MDQ0ZTBmZTUifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a02829f6fa04475b687e4c&resolution=resolved)

ivyleavedtoadflax commented:
Something for you and me i think @nsorros, let's come up with a way to categorize better ![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNhMDA1YWI0MTgwNDcwMDA5YTk0ZDk2IiwiaWQiOiI0OTI3ODcyYy05NTMyLTRhY2UtOTk2YS00NWU1ODBhMjQ1YzQifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /work/
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2Fwork%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/work/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a0285f2b35ad4419e483b4&resolution=resolved)

ivyleavedtoadflax commented:
Let's update this

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNhMDA1YWI0MTgwNDcwMDA5YTk0ZDk2IiwiaWQiOiI4NzM4MTk2Zi0xN2I2LTQxMTUtODQ4Yy1iMjdiZDU5YjMzYmMifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /about/
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2Fabout%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/about/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a02e3dc1082f4e59d1e5bc&resolution=resolved)

nsorros commented on README.md:
```diff
@@ -4,3 +4,51 @@ We are Mantis,
+
+## Add Blog Items
+
+All blog data are fecthing from medium. However, we can also add our own blog items locally.
```
Does this happen automatically at build?

kausarpial replied:
Yes, it'll happen automatically at the build. 
If you publish a new article on medium, just rebuild the website from netlify. It'll automatically fetch the new blog for you.

nsorros replied:
This is super 👍

kausarpial commented:
> > Hi @nsorros @ivyleavedtoadflax I've made some changes on netlify to build, and also updated the readme file. Now, the deploy preview will work.
> > Let me know of further changes. Thanks
> 
> We need to remove the first item in work and change the title of the second one according to whats live atm. Should I try to do the change for practice?
> 
> ![screenshot](https://camo.githubusercontent.com/0671a9c6cf2c0103083df14e3e373b52f64810461b7d76c571b3592f1d507477/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f695a44557a5a6d45324f4449745a546b305a4330304f4449344c546b794e4467745a57466a4e6d5a684e4745314d6a5a68496977695a4756776247393558326c6b496a6f694e6a4e684d444131595749304d5467774e4463774d44413559546b305a446b3249697769615751694f6949304e5751774d5467314e5330334e6a51344c54526b4e6d5574596a566c4e43316a4f544535596a4a6959544d7a4e54556966513d3d)

Yeah, Sure. Please go ahead. 
Let me know of any problem. 👍 
@nsorros

nsorros commented:
> Let's update this
> 
> ![screenshot](https://camo.githubusercontent.com/e80dc797af8207abcd9bd710fcb19ef80f635951ab78df2e6a55d62a722b2d69/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f695a44557a5a6d45324f4449745a546b305a4330304f4449344c546b794e4467745a57466a4e6d5a684e4745314d6a5a68496977695a4756776247393558326c6b496a6f694e6a4e684d444131595749304d5467774e4463774d44413559546b305a446b3249697769615751694f6949344e7a4d344d546b325a6930784e3249324c5451784d5455744f445134597931694d6a64695a445535596a4d7a596d4d6966513d3d)
> Browser metadata
> 
> [Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/about/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a02e3dc1082f4e59d1e5bc&resolution=resolved)

if you mean to add Daniel and Chris, we have another card for this but we could also do here, why not.

nsorros commented:
> More a content question, but perhaps we should change this to be more general - less medical specific @nsorros ![screenshot](https://camo.githubusercontent.com/ea769f226797215ac472593e20e6be259bda7d4ba4153e2488f7d9d04e048191/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f695a44557a5a6d45324f4449745a546b305a4330304f4449344c546b794e4467745a57466a4e6d5a684e4745314d6a5a68496977695a4756776247393558326c6b496a6f694e6a4e684d444131595749304d5467774e4463774d44413559546b305a446b3249697769615751694f694a694d4751784f4463794d5330334d446b304c54526d5a5755744f545a685a5330314d6a63344d4451305a54426d5a54556966513d3d)
> Browser metadata
> 
> [Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63a005ab4180470009a94d96&commentID=63a02829f6fa04475b687e4c&resolution=resolved)

When did we add this section?

nsorros commented:
Hey @kausarpial, where can I find the markdown or html for the home page? I dont see it in content? Also in the about section, I only see the beliefs in the markdown, how about the other sections? I would like to add more members of the team for example.

kausarpial commented:
> Hey @kausarpial, where can I find the markdown or html for the home page? I dont see it in content? Also in the about section, I only see the beliefs in the markdown, how about the other sections? I would like to add more members of the team for example.
> 
> ## Update
> I noticed that the rest of the about is in `public/about/index.html`, can we move the content in markdown? Similarly for the home page that seems to be in `public/index.html`. We want the content to be in markdown to make it easier to update it, this is one of the main reasons we went with hugo.

Hi Nick,
We don't have to change the `public` directory to make the changes to our website. `public` directory will autogenerate the `HTML` codes based on the content. 
I'm sending a loom record to help you with that. 
Hope that'll help to understand the concept of how Hugo works. 
And everything is documented on our `modules`. here's the link for that: https://deploy-preview-67--mantisnlp.netlify.app/modules/

Loom: https://www.loom.com/share/9f68329dc6064833b6fd7f91f836ff17

nsorros commented:
> Hi Nick, We don't have to change the `public` directory to make the changes to our website. `public` directory will autogenerate the `HTML` codes based on the content. I'm sending a loom record to help you with that. Hope that'll help to understand the concept of how Hugo works. And everything is documented on our `modules`. here's the link for that: https://deploy-preview-67--mantisnlp.netlify.app/modules/
> 
> Loom: https://www.loom.com/share/9f68329dc6064833b6fd7f91f836ff17

Super, I added Daniel. Note that in the new website we added a section about collaborators https://mantisnlp.com/about.html. Could you tell me whats the best way to add this? This is a good opportunity to learn how we can extend the website a bit beyond its design so let me know via loom otherwise how i can address this change.

nsorros commented:
@ivyleavedtoadflax have another review

kausarpial commented:
> > Hi Nick, We don't have to change the `public` directory to make the changes to our website. `public` directory will autogenerate the `HTML` codes based on the content. I'm sending a loom record to help you with that. Hope that'll help to understand the concept of how Hugo works. And everything is documented on our `modules`. here's the link for that: https://deploy-preview-67--mantisnlp.netlify.app/modules/
> > Loom: https://www.loom.com/share/9f68329dc6064833b6fd7f91f836ff17
> 
> Super, I added Daniel. Note that in the new website we added a section about collaborators https://mantisnlp.com/about.html. Could you tell me whats the best way to add this? This is a good opportunity to learn how we can extend the website a bit beyond its design so let me know via loom otherwise how i can address this change @kausarpial

Hi @nsorros 
Here's the loom to get the idea about adding the collaborators' section: https://www.loom.com/share/b6305567eaba45ee94969aa9638daa58

Thanks. Let me know of further requests.

ivyleavedtoadflax commented:
It would be good to change this / resize

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNjNTBkYjExNzZmNjEwMDA4ZDI3OGRmIiwiaWQiOiJjMzE3YzY3Yi0zMTJiLTRjYzEtYmE4Mi00Y2RjYzk1ZjI0NzcifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63c50db1176f610008d278df&commentID=63c50fb1035bd310a571634b&resolution=resolved)

ivyleavedtoadflax commented:
We've lost the links to other pages here, is that intended?

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNjNTBkYjExNzZmNjEwMDA4ZDI3OGRmIiwiaWQiOiIwYmI1NDkxYy02MDk1LTQ4ZmItYjMzMy0yNzZkMWY5NWFkZDYifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63c50db1176f610008d278df&commentID=63c50ffd035bd31085716397&resolution=resolved)

ivyleavedtoadflax commented:
☝ I'm guessing not, as I now can't access the about page/ case studies
<details><summary>Browser metadata</summary>





```
Path:      /#cta
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2F%23cta&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/#cta) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63c50db1176f610008d278df&commentID=63c5103aae6b2017b2ffba18&resolution=resolved)

nsorros commented:
> We've lost the links to other pages here, is that intended?
> 
> ![screenshot](https://camo.githubusercontent.com/da363a1eebb5ff50f7890e7630e270cca4e07e23ae934f7510d5ed64e791d570/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f695a44557a5a6d45324f4449745a546b305a4330304f4449344c546b794e4467745a57466a4e6d5a684e4745314d6a5a68496977695a4756776247393558326c6b496a6f694e6a4e6a4e54426b596a45784e7a5a6d4e6a45774d4441345a4449334f47526d49697769615751694f694977596d49314e446b78597930324d446b314c5451345a6d4974596a4d7a4d7930794e7a5a6b4d5759354e57466b5a44596966513d3d)
> Browser metadata
> 
> [Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63c50db1176f610008d278df&commentID=63c50ffd035bd31085716397&resolution=resolved)

Was just about to report this @kausarpial, I noticed the header dissapeared. This is not happening locally, can you investigate?

Another issue is, I added a new case study that has a non square badge and its not displaying properly. Additionally the background is transparent. Is there any way we can account for that. I have commented out the badge but you can see the problem is you uncomment it.

kausarpial commented:
> > We've lost the links to other pages here, is that intended?
> > ![screenshot](https://camo.githubusercontent.com/da363a1eebb5ff50f7890e7630e270cca4e07e23ae934f7510d5ed64e791d570/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f695a44557a5a6d45324f4449745a546b305a4330304f4449344c546b794e4467745a57466a4e6d5a684e4745314d6a5a68496977695a4756776247393558326c6b496a6f694e6a4e6a4e54426b596a45784e7a5a6d4e6a45774d4441345a4449334f47526d49697769615751694f694977596d49314e446b78597930324d446b314c5451345a6d4974596a4d7a4d7930794e7a5a6b4d5759354e57466b5a44596966513d3d)
> > Browser metadata
> > [Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63c50db1176f610008d278df&commentID=63c50ffd035bd31085716397&resolution=resolved)
> 
> Was just about to report this @kausarpial, I noticed the header dissapeared. This is not happening locally, can you investigate?
> 
> Another issue is, I added a new case study that has a non square badge and its not displaying properly. Additionally the background is transparent. Is there any way we can account for that. I have commented out the badge but you can see the problem is you uncomment it.

Hi @nsorros @ivyleavedtoadflax 

Resolved the issues mentioned above, can you have a look please? 
Hopefully we can go the live now.
Also a soft reminder before merge into master, we need to change the `base URL` and `calendly meeting URL` .

nsorros commented:
Let's get @ivyleavedtoadflax to add a last case study and review the changes and we should be ready to go. Almost there. Great work @kausarpial

ivyleavedtoadflax commented:
I need to fix the path for this image, then i think we are good to go.

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNjZTg3NTkxOGJlNGEwMDA4OTA1NmUzIiwiaWQiOiJmY2U0ZWM4Yy1lNDI2LTRhODAtODUzMi0xNGJkZWNlMDdmZWEifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /work/
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1916 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2Fwork%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1916x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/work/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63ce875918be4a00089056e3&commentID=63d254ecddaccd336adbce80&resolution=resolved)

ivyleavedtoadflax commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNjZTg3NTkxOGJlNGEwMDA4OTA1NmUzIiwiaWQiOiI2YjdjOTc1Yi01ZjE0LTQxYzAtYTZlNy0wOTc2MDgwOGI5M2MifQ==)

This is lorem, not the real text. @nsorros the real text needs to be added to these other case studies
<details><summary>Browser metadata</summary>





```
Path:      /work/granted-themes/
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  1920 x 975 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2Fwork%2Fgranted-themes%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=1920x975&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/work/granted-themes/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63ce875918be4a00089056e3&commentID=63d40d8c200e2d2265085a11&resolution=resolved)

ivyleavedtoadflax commented:
I think this is out of place

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDUzZmE2ODItZTk0ZC00ODI4LTkyNDgtZWFjNmZhNGE1MjZhIiwiZGVwbG95X2lkIjoiNjNjZTg3NTkxOGJlNGEwMDA4OTA1NmUzIiwiaWQiOiI5MGVmN2NhNy0wOWE4LTQ2ZDMtYjExMy0wYjQ3NjAwZGRiYjAifQ==)


<details><summary>Browser metadata</summary>





```
Path:      /work/overton/
Browser:   Chrome 108.0.0.0 on Linux x86_64
Viewport:  956 x 924 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2Fwork%2Foverton%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=108.0&resolution=956x924&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/work/overton/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63ce875918be4a00089056e3&commentID=63d40fd28689eb1f9fdb4e8e&resolution=resolved)

ivyleavedtoadflax commented:
this looks good. Let's pull the trigger. Something to add soon will be the privacy policy
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 109.0.0.0 on Linux x86_64
Viewport:  1920 x 975 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-67--mantisnlp.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=109.0&resolution=1920x975&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-67--mantisnlp.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63db6fff33754500080e7903&commentID=63db99b002d61c3549740907&resolution=resolved)

ivyleavedtoadflax reviewed (approved):
LGTM

nsorros commented:
@kausarpial we are ready to merge, what steps do we need to do before / after merging?

kausarpial commented on config.toml:
```diff
@@ -0,0 +1,41 @@
+baseURL = 'https://deploy-preview-67--mantisnlp.netlify.app/'
```
We need to change our base URL to `https://mantisnlp.com/` before going to merge

kausarpial commented on data/callToAction.yml:
```diff
@@ -0,0 +1,9 @@
+  Do you have a Natural Language Processing problem you need help with?
+callToActionnText: "Let's Talk"
+callToActionLink: "/contact"
+calendlyLink: "https://calendly.com/mojo-customer-success-1/tips-tricks-webinar"
```
Also, we need to change this calendly events to our calendly event link.
@nsorros

ivyleavedtoadflax replied:
resolved in https://github.com/MantisAI/mantisnlp.com/pull/67/commits/d012f3ddac661c008db8cb2636c56a44f90a3899

ivyleavedtoadflax commented:
It's Friday evening so I'm going to hold off merging, but I think this is good to go :rocket:
